backend/utils/user_session.py

```diff
@@ -34,3 +36,27 @@ def set_organization_member_role(
     @staticmethod
     def get_organization_member_role(request: HttpRequest) -> Optional[str]:
         return request.session.get("role")
+
+    @classmethod
+    def is_authorized_path(cls, request: HttpRequest) -> bool:
+        """Checks if the requested path is authorized based on the organization
+        ID of user.
+
+        Args:
+            request (HttpRequest): The HTTP request containing session data,
+            including the organization ID.
+
+        Returns:
+            bool: `True` if the path is authorized; `False` otherwise.
+        """
+
+        organization_uid = connection.tenant.organization_id
```

Comment:
@muhammad-ali-e can we name it as `requested_organization_uid` to make more sense?